MushroomObserver/mushroom-observer
# Add ml flag

To test:

1. Go to a show image page
2. See the "Okay for Machine Learning | Not for Machine Learning" UI elements
3. Ensure that they work and are persistent
